components/dashboard/member-list.tsx

```
@@ -0,0 +1,205 @@
+							</TooltipTrigger>
+						<TooltipContent>Toggle sort direction</TooltipContent>
+					</Tooltip>
+					<Tooltip>
+							<TooltipTrigger asChild>
+								<Button outline className="h-8 px-3 text-sm" onClick={exportCsv} aria-label="Export CSV"><IconDownload className="size-4" /></Button>
+							</TooltipTrigger>
+						<TooltipContent>Export CSV</TooltipContent>
+					</Tooltip>
+				</div>
+			</div>
+			<div className="glass-panel overflow-hidden" data-elev={2} role="table" aria-label="Member share allocations">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<table>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)